src/main/java/org/tron/core/net/node/NodeDelegateImpl.java

```diff
@@ -149,18 +149,18 @@ public boolean handleTransaction(TransactionCapsule trx) throws BadTransactionEx
       logger.info("AccountResourceInsufficientException" + e.getMessage());
       return false;
     } catch (DupTransactionException e) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Move this left curly brace to the beginning of next line of code. ([squid:LeftCurlyBraceStartLineCheck](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1500&issues=AWXBxdl6ashydt7ke7DN&open=AWXBxdl6ashydt7ke7DN)

[ISSUE_KEY]: AWXBxdl6ashydt7ke7DN